doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -814,35 +817,40 @@ To delete a key, follow these steps.
 
 ## 6 X-Road Services
 
-The services are managed on two levels:
+X-Road supports both SOAP and REST services. The services are managed on two levels:
 
--   the addition, deletion, and deactivation of services is carried out on the WSDL/REST level;
+-   the addition, deletion, and deactivation of services is carried out on the WSDL / REST API level;
 
--   the service address, internal network connection method, and the service timeout values are configured at the service level. However, for WSDL, it is easy to extend the configuration of one service to all the other services.
+-   the service address, internal network connection method, and the service timeout values are configured at the service level for SOAP services and at the API level for REST services. In addition, for SOAP / WSDL, it is easy to extend the configuration of one service to all the other services.
 
 
 ### 6.1 Adding a service description
 
 **Access rights:** [Service Administrator](#xroad-service-administrator)
 
+### 6.1.1 SOAP
+
 When a new WSDL file is added, the security server reads service information from it and displays the information in the table of services. The service code, title and address are read from the WSDL.
-When a new REST service is added, the security server displays url and service code provided.
 
 **To add a WSDL**, follow these steps.
 
 1.  On the **Configuration** menu, select **Security Server Clients**, select a client from the table and click the **Services** icon on that row.
 
-2.  Click **ADD WSDL**, enter the WSDL address in the window that opens and click **OK**. The WSDL and the information about the services it contains are added to the table. By default, the WSDL is added in disabled state (see [6.3](#63-enabling-and-disabling-a-wsdl)).
+2.  Click **ADD WSDL**, enter the WSDL address in the window that opens and click **OK**. The WSDL and the information about the services it contains are added to the table. By default, the WSDL is added in disabled state (see [6.3](#63-enabling-and-disabling-a-service-description)).
 
 **To see a list of services contained in the WSDL**
 
 -   click the “**+**” symbol in front of the WSDL row to expand the list.
 
+### 6.1.2 REST
+
+When a new REST service is added, the security server displays url and service code provided.
+
 **To add a REST service**, follow these steps.
 
 1.  On the **Configuration** menu, select **Security Server Clients**, select a client from the table and click the **Services** icon on that row.
 
-2.  Click **ADD REST**, enter the url and service code in the window that opens and click **OK**.  
+2.  Click **ADD REST**, enter the url and service code in the window that opens and click **OK**. The url and the service code are added to the table. By default, the REST API is added in disabled state (see [6.3](#63-enabling-and-disabling-a-service-description)).
 
 **To see the service the REST service**
```

Comment:
This sentence should be rephrased somehow

Reply:
Did a minor update to the sentence. Originally, it was copied from the SOAP service instructions that have exactly the same structure and format.